MastodonSDK/Sources/CoreDataStack/Entity/Mastodon/StatusEdit.swift

```diff
@@ -31,6 +31,9 @@ public final class StatusEdit: NSManagedObject {
     // sourcery: autoUpdatableObject, autoGenerateProperty
     @NSManaged public var spoilerText: String?
 
+    // sourcery: autoUpdatableObject, autoGenerateProperty
+    @NSManaged public var status: Status?
```

Comment:
❓ Does the CoreData model already reflect this relationship? I haven't seen any changes to our Model in this PR.

Reply:
Yes:

https://github.com/mastodon/mastodon-ios/blob/d24fe00420849f008a3301a754f7d601fdcf63e3/MastodonSDK/Sources/CoreDataStack/CoreData.xcdatamodeld/CoreData%208.xcdatamodel/contents#L255